tests/integration/test_charm.py

```diff
@@ -162,11 +164,14 @@ async def check_alert_propagation(url, alert_name):
     assert alert_rule is not None and alert_rule["state"] == "firing"
 
 
+@pytest.mark.abort_on_fail
 @pytest.mark.asyncio
 async def test_seldon_alert_rules(ops_test: OpsTest):
     """Test Seldon alert rules."""
     # NOTE: This test is re-using deployments created in test_build_and_deploy()
-    namespace = ops_test.model_name
+    # Use namespace "default" to create seldon deployments
+    # due to https://github.com/canonical/seldon-core-operator/issues/218
+    namespace = DEFAULT_NAMESPACE
```

Comment:
Am on the fence on dropping the `namespace` var and use the `WORKLOAD_NAMESPACE` directly, to slightly simplify the code. But will let it up to you

Reply:
Thanks for the suggestion, I think it's fine though and we can keep it as is.